flocker/node/agents/test/blockdevicefactory.py

```diff
@@ -54,6 +52,7 @@ class ProviderType(Names):
     """
     openstack = NamedConstant()
     aws = NamedConstant()
+    rackspace = NamedConstant()
```

Comment:
`openstack` and `rackspace` don't seem like the same kinds of things to me.  If `rackspace` is how you do Rackspace, what is `openstack` for now?